docs/book/src/design/wallet-tee.md

```diff
@@ -0,0 +1,25 @@
+# Wallet TEE
+
+The Wallet TEE consists of a trusted SGX enclave, alongside an untrusted host service.
+
+## SGX enclave
+
+The Wallet SGX enclave contains the system's [Trusted Computing Base (TCB)][TCB], and is responsible for all security-sensitive wallet operations:
+
+- Generating and managing account keypairs
+- Performing privileged operations such as transaction signing
+
+Once generated, secret keys and other sensitive data never leaves the enclave unencrypted. All persistent storage is cryptographically sealed and authenticated, and all trusted communication with the enclave occurs over cryptographically attested and mutually-authenticated channels.
```

Comment:
As it is specific to the context of TEEs, the concept of *sealed storage* may need defining

Reply:
Yup, this can probably go in the glossary alongside other SGX concepts.

Reply:
Ah yes, I concur